backend/routes/employees.js

```diff
@@ -31,6 +32,10 @@ router.get('/:id', getEmployee)  // Where :id is a router parameter
 // POST employee
 router.post('/', addEmployee) // Where :id is a router parameter
```

Comment:
Dumb comment, there's no :id parameter in this function

Reply:
Done.